python/sglang/srt/layers/layernorm.py

```diff
@@ -100,6 +102,14 @@ def forward_cuda(
     ) -> Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]]:
         if self.variance_size_override is not None:
             return self.forward_native(x, residual)
+        if is_batch_invariant_mode_enabled():
+            if residual is not None:
```

Comment:
maybe when `get_global_server_args().rl_on_policy_target == "fsdp"` also forcefully call native fn now to avoid breaking true on policy on slime. I will enable the flag and check whether computation is compatible and also handle residual-is-not-none in sync w/ slime.